feconf.py

```diff
@@ -426,6 +426,8 @@ def get_empty_ratings():
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_MESSAGE_EMAIL_HANDLER_URL = (
     '%s/feedbackemailhandler' % EMAILS_TASK_PREFIX)
+FEEDBACK_MESSAGE_EMAIL_CLEAR_URL = (
```

Comment:
(Also I don't think this has anything to do with the task itself, so no need for EMAILS_TASK_PREFIX)